languages/common-lisp/exercises/concept/lillys-lasagna-leftovers/lillys-lasagna-leftovers-test.lisp

```diff
@@ -0,0 +1,51 @@
+;; Ensures that lillys-lasagna-leftovers.lisp and the testing library are always loaded
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  (load "lillys-lasagna-leftovers")
+  (ql:quickload :fiveam))
+
+;; Defines the testing package with symbols from lillys-lasagna-leftovers and FiveAM in scope
+;; The `run-tests` function is exported for use by both the user and test-runner
+(defpackage :lillys-lasagna-leftovers-test
+  (:use :cl :fiveam :lillys-lasagna-leftovers)
+  (:export :run-tests))
+
+;; Enter the testing package
+(in-package :lillys-lasagna-leftovers-test)
+
+;; Define and enter a new FiveAM test-suite
+(def-suite lillys-lasagna-leftovers-suite)
+(in-suite lillys-lasagna-leftovers-suite)
+
+(test preparation-time
```

Comment:
I think we should start adding test descriptions! We can just do something like `(test preparation-time "A description of the expected behaviour or something" ...`. The test-runner should then be able to use those as test-names :smile:

Reply:
I'll look into doing that... thanks for the idea.

Reply:
From the documentation `5am:test` does not take a description. It takes a name and body. The Name can be a list of name, depends-on and suite.

Reply:
Huh, you are definitely right about this! That caught me a bit off guard! With that being said, the macro is written in a way such that it checks if the first form of the body is a string, then sets it as a description if so. From the source [here](https://common-lisp.net/project/fiveam/docs/api/macro_005FIT.BESE.FIVEAM_003A_003ATEST.html):

```lisp
(setf description (if (stringp (car body))
                      (pop body)
                       "")
```

As a double-check of this, the pal-picker exercise has these descriptions and the test-runner uses those as the "name" field without any issues. :)

Reply:
@verdammelt [Here is the exercise](https://github.com/exercism/v3/blob/master/languages/common-lisp/exercises/concept/pal-picker/pal-picker-test.lisp) where I have these in use :slightly_smiling_face: